test/protocol-tests/src/main/resources/codegen-resources/restjson/contenttype/endpoint-tests.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Why are these empty  ? Can we default to empty if File[xml/endpoint-tests.json] doesnot exist ?

Reply:
Empty because these are just testing clients. For real service modules, we want them to exist to ensure that the the providers are being tested.